package.json

```diff
@@ -40,31 +40,30 @@
     "semantic-release": "semantic-release"
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^4.21.0"
+    "@typescript-eslint/experimental-utils": "^4.24.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "^12.1.1",
-    "@commitlint/config-conventional": "^12.1.1",
-    "@types/jest": "^26.0.22",
-    "@typescript-eslint/eslint-plugin": "^4.21.0",
-    "@typescript-eslint/parser": "^4.21.0",
+    "@commitlint/cli": "^12.1.4",
+    "@commitlint/config-conventional": "^12.1.4",
+    "@types/jest": "^26.0.23",
+    "@typescript-eslint/eslint-plugin": "^4.24.0",
+    "@typescript-eslint/parser": "^4.24.0",
     "cpy-cli": "^3.1.1",
-    "eslint": "^7.26.0",
+    "eslint": "^7.27.0",
     "eslint-config-kentcdodds": "^18.1.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.23.2",
+    "eslint-plugin-import": "^2.23.3",
     "eslint-plugin-jest": "^24.3.6",
-    "eslint-plugin-jest-formatting": "^2.0.1",
+    "eslint-plugin-jest-formatting": "^3.0.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-promise": "^5.1.0",
     "husky": "^4.3.8",
     "jest": "^26.6.3",
-    "jest-environment-jsdom": "^25.5.0",
```

Comment:
Oh, so this is not necessary anymore. I don't know why, but great news.

Reply:
Yeah I figured now we have tests running on Node v16, that we could remove it

Reply:
This was actually failing for node v10 😅

Reply:
Than I don't know what was causing the issue 😅

But it's fixed so you don't hear me complaining 😉